### DISCUSSION TITLE
Duration columns

### DISCUSSION THREAD

**Discussion by lmartins30** [CamDavidsonPilon/lifelines#1555] [Q&A]
I'm trying to fit 2 COXPH models with 2 different duration columns. Is it possible to make one model with 2 duration_col on fit?

**Comment by CamDavidsonPilon** [comment]
Hi @lmartins30,

I would suggest just creating two `CoxPHFitter` objects. Any reason not to do this?

**Reply by lmartins30** [reply]
The model I made is not working properly, the results are quite bad. I wanted to try with 2 columns at same to time to restrict more because I have just 1% of events on the dataset.
I've already tried 2 different, taking out the other one but no considerable improvement was observed.
This is just and idea I had but been trying another that were tested and validated.
Thanks
